projects/components/src/divider/divider.component.scss

```diff
@@ -2,6 +2,5 @@
 
 .divider {
   width: 100%;
-  border: 1px solid $gray-2;
-  border-radius: 1px;
+  border-bottom: 1px solid $gray-2;
```

Comment:
are you sure?

Reply:
I'm sure this makes it 1px. Not positive we don't have use cases for 2px, but the ones I know of are 1px. Not a huge deal either way.